logos/logos-ui/src/app/core/layout/shell/shell.scss

```
@@ -6,13 +6,7 @@
   height: 100vh;
   overflow: hidden;
 
-  background:
-    linear-gradient(
-      135deg,
-      rgb(var(--shell-gradient-start)) 0%,
-      rgb(var(--shell-gradient-mid)) 45%,
-      rgb(var(--shell-gradient-end)) 100%
-    );
+  background: rgb(var(--shell-background));
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)